### PR TITLE
Feature/tcvp 1710

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- JIRA tickets 1710, 1711, 1712 & some suggestions to simplify our email sending e.g. removing cc/bcc support

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Run new methods for adding a file history record and retrieving by ticket number

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .gitops/charts/traffic-court-dev-values.yaml [hunk 1]
```diff
@@ -7,14 +7,14 @@ global:
 
 arc-dispute-api:
   image:
-    tag: "1.44.4"
+    tag: "1.44.5"
```
Comment (KDNeufeld) [inline .gitops/charts/traffic-court-dev-values.yaml:10]: This is causing merge conflicts. In the future, I think it'd be better to separate release commits and application commits.  The changes in this file should be in a separate PR.

**Review by KDNeufeld — APPROVED** [review]
Great work!

File: src/backend/TrafficCourts/Messaging/MessageContracts/SendEmail.cs [hunk 2]
```diff
@@ -11,11 +12,12 @@ namespace TrafficCourts.Messaging.MessageContracts
     /// </summary>
     public class SendEmail : IMessage
     {
-        public string? From { get; set; }
-        public IList<string> To { get; set; } = new List<string>();
+        public string? FromEmailAddress { get; set; }
```
[the remark this answers is not in the document]
Comment (lfdgov) [reply src/backend/TrafficCourts/Messaging/MessageContracts/SendEmail.cs:15]: This was causing an issue I believe with the Java side of things with compilation.  I wanted to keep the same name throughout so changed it.

File: src/backend/TrafficCourts/Messaging/MessageContracts/SendEmail.cs [hunk 3]
```diff
@@ -11,11 +12,12 @@ namespace TrafficCourts.Messaging.MessageContracts
...
+        public string? ToEmailAddress { get; set; } 
         public string Subject { get; set; } = String.Empty;
         public string? PlainTextContent { get; set; }
         public string? HtmlContent { get; set; }
         public string TicketNumber { get; set; } = String.Empty;
+        public EmailHistorySuccessfullySent SuccessfullySent;
```
[the remark this answers is not in the document]
Comment (lfdgov) [reply src/backend/TrafficCourts/Messaging/MessageContracts/SendEmail.cs:21]: I will put a meeting on our calendars

**Pull request closed without merging**